<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查找与替换</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>将光标选中的区域替换为：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>查找以下内容：</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="nextBtn">
     <property name="text">
      <string>查找(在下一个位置)</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="resLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="afterTextEdit"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="exitBtn">
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="replaceButton">
     <property name="text">
      <string>替换</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
